doc/distrib/NodeHelpFiles/_review/Revit.Elements.TextNote.SetText.md

```diff
@@ -0,0 +1,9 @@
+## In Depth
+`TextNote.SetText` sets the given text note to the given value.
```

Comment:
Rephrase for clarity: TextNote.SetText sets the given text value to the text note.